src/in-house-liquidity/index.test.ts

```diff
@@ -54,4 +65,217 @@ describe('In House Liquidity Calls', () => {
       ).toThrow('Cannot call IHL because walletAddress is null')
     })
   })
+
+  describe('makeRequest', () => {
+    it('makes a FC authenticated call', async () => {
```

Comment:
Can you also include `if provider info is passed / not passed` to the test name? Wasn't super obvious when reading these tests.
Also, should these tests also assert that the function returns correct response?